UPGRADING.md

```diff
@@ -67,6 +67,21 @@ the correct code.
 
 ### Modules
 
+#### `**all**`
```

Comment:
can we update the docs on events. will also need to update tutorials

Reply:
Right, done! I have mentioned the change to Xavier too.